src/js/media-chrome-range.js

```diff
@@ -209,6 +211,8 @@ class MediaChromeRange extends window.HTMLElement {
     this.container = this.shadowRoot.querySelector('#container');
     this.range = this.shadowRoot.querySelector('#range');
     this.range.addEventListener('input', this.updateBar.bind(this));
+
+    this.#thumbWidth = parseInt(getComputedStyle(this).getPropertyValue('--media-range-thumb-width') || 10);
```

Comment:
nit: should add a radix to not parse incorrectly
```suggestion
    this.#thumbWidth = parseInt(getComputedStyle(this).getPropertyValue('--media-range-thumb-width') || 10, 10);
```